mem0/memory/storage.py

```diff
@@ -139,6 +139,20 @@ def get_history(self, memory_id):
             for row in rows
         ]
 
+    def search_history(self, new_memory):
+        memory_list = [s.strip() for s in new_memory.split("- ") if s]
```

Comment:
@NavyaAlapati13 why are you splitting on "- "?

Reply:
That's because the extracted memory string being passed had "- " in the beginning, 
which was not letting the SQL query fetch existing data.

Reply:
We did not explicitly ask the model to return the output memory with "-" at the start. I do not see any drawbacks to your implementation.

Reply:
So, this implementation is fine, right?